package.json

```diff
@@ -16,5 +16,8 @@
     "start-backend": "cd scripts && \"./startBackendWindows1.sh\"",
     "finish-backend": "cd scripts && \"./startBackendWindows2.sh\"",
     "frontend": "cd scripts && \"./startFrontend.sh\""
+  },
+  "dependencies": {
+    "axios": "^0.19.2"
```

Comment:
Axios is a good idea, however here you are in the root folder, we do not add any dependencies here,  the dependency is needed for **frontend** so you would run `npm install`  command inside /frontend folder instead, not here. So I would remove it from here

```suggestion
```